api_core/google/api_core/datetime_helpers.py

```diff
@@ -115,45 +116,22 @@ def from_iso8601_time(value):
 
 
 def from_rfc3339(value):
-    """Convert a microsecond-precision timestamp to datetime.
-
-    Args:
-        value (str): The RFC3339 string to convert.
-
-    Returns:
-        datetime.datetime: The datetime object equivalent to the timestamp in
-            UTC.
-    """
-    return datetime.datetime.strptime(value, _RFC3339_MICROS).replace(tzinfo=pytz.utc)
-
-
-def from_rfc3339_nanos(value):
-    """Convert a nanosecond-precision timestamp to a native datetime.
+    """Convert a nanosecond-precision or if nanoseconds are missing
+    microsecond-precision timestamp to a native datetime.
 
     .. note::
         Python datetimes do not support nanosecond precision; this function
         therefore truncates such values to microseconds.
 
     Args:
-        value (str): The RFC3339 string to convert.
+        value (str): The RFC 3339 string to convert.
```

Comment:
We should be consistent with RFC3339 or RFC 3339.  I propose `RFC3339` as it matches the method name spacing.